### PR TITLE
EV-10240 overrides baseuri from env if set

### PR DESCRIPTION
**Pull request by hnrchrdl**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const defaultOptions = {
     baseUri: '',
     extensions: ['.gif', '.jpeg', '.jpg', '.png', '.svg'],
     flatten: false,
+    envName: 'ASSET_BASE_URI'
```
Comment (flecno) [inline src/index.js]: I would call that omething else.

```suggestion
    envName: 'ASSET_BASE_URI'
    baseUriFrom: 'ASSET_BASE_URI'
```

**Review by flecno — COMMENTED** [review]
I don't know much about babel plugins. I think a small test is missing and the existing tests are failing

File: src/index.js (hunk 1 above)
Comment (hnrchrdl) [reply src/index.js]: I should have looked at the tests. should be fine now.

**Comment by hnrchrdl** [comment]
project is now running pipeline tests for pull requests and master.

**Review by flecno — APPROVED** [review]

**Pull request merged**